Core/src/Geometry/CylinderVolumeBounds.cpp

```diff
@@ -48,6 +49,7 @@ Acts::CylinderVolumeBounds::CylinderVolumeBounds(
   m_values[eHalfLengthZ] = 0.5 * thickness;
   m_values[eHalfPhiSector] = rBounds.get(RadialBounds::eHalfPhiSector);
   m_values[eAveragePhi] = rBounds.get(RadialBounds::eAveragePhi);
+  m_values[eAngle] = rBounds.get(RadialBounds::eAngle);
   buildSurfaceBounds();
 }
 
```

Comment:
In ATLAS we have solved this with `BevelledCylinderVolumeBounds` in order to not have too many if statements ... not sure what's the better solution here.

Reply:
Let's keep it though.

Reply:
> Hi @noraemi - one quick question: can the bevelling also be in the same direction on both sides?
> 
> The ATHENA-EIC guys need a tilted beam pipe inlay into a cylinder volume (we can specify that volume shape later), but that would need some sort of parallel cut off on both sides.

Yes! the bevel can be done in both directions e.g: \\___    or    /___
the two sides are independent as well. Will add a comment about what the bevel is w.r.t somewhere in the CylinderBounds

Reply:
Hmm, I think I forgot this. In anycase, wouldn't this mean quite a lot of duplicated code with minor additions? But perhaps that's the preferred option..